src/libs/search/filters/index.ts

```diff
@@ -1,17 +1,28 @@
 import { SearchOptions } from '@elastic/app-search-javascript'
 
+import { CategoryNameEnum } from 'api/gen'
 import { SearchState } from 'features/search/types'
 import { GeoCoordinates } from 'libs/geolocation'
 import { buildBoosts } from 'libs/search/filters/buildBoosts'
 
 import { buildFacetFilters } from './buildFacetFilters'
 import { buildGeolocationFilter } from './buildGeolocationFilter'
 import { buildNumericFilters } from './buildNumericFilters'
-import { AppSearchFields, RESULT_FIELDS, SORT_OPTIONS } from './constants'
+import { AppSearchFields, FALSE, RESULT_FIELDS, SORT_OPTIONS } from './constants'
+
+// filter out non free digital offers except press category
```

Comment:
Tu peux expliciter le commentaire comme ça:
// we don't want to show offers to underage users if those offers are digital, not free and not in press category.
// As a result if an offer is either not digital, free, or is in category press, we can show to the underage users.